docs/getting_started.md

```diff
@@ -0,0 +1,67 @@
+
+## Installation
+
+For installation, multiple options are available depending on your setup:
+
+=== "Overview"
+
+    | Source | Status |
```

Comment:
I like the content tabs, but I'd limit them purely to the code block. With the surrounding text, it's not clear until where the scope of the tab applies

Reply:
Ok! I've modified this